ERROR - 2022-12-14 17:13:32 --> mysqli_sql_exception: Unknown column 'users.deleted_at' in 'where clause' in C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Model.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\BaseModel.php(524): CodeIgniter\Model->doFind(true, '1')
#6 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Authentication\AuthenticationBase.php(130): CodeIgniter\BaseModel->find('1')
#7 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
#8 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Controllers\AuthController.php(107): Myth\Auth\Authentication\LocalAuthenticator->check()
#9 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\CodeIgniter.php(927): Myth\Auth\Controllers\AuthController->logout()
#10 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#11 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp7\htdocs\regsosek1206\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp7\\htdoc...')
#14 {main}
CRITICAL - 2022-12-14 17:13:32 --> Call to a member function getFirstRow() on bool
in SYSTEMPATH\Model.php on line 183.
 1 SYSTEMPATH\BaseModel.php(524): CodeIgniter\Model->doFind(true, '1')
 2 VENDORPATH\myth\auth\src\Authentication\AuthenticationBase.php(130): CodeIgniter\BaseModel->find('1')
 3 VENDORPATH\myth\auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
 4 VENDORPATH\myth\auth\src\Controllers\AuthController.php(107): Myth\Auth\Authentication\LocalAuthenticator->check()
 5 SYSTEMPATH\CodeIgniter.php(927): Myth\Auth\Controllers\AuthController->logout()
 6 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 7 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
ERROR - 2022-12-14 17:13:36 --> mysqli_sql_exception: Unknown column 'users.deleted_at' in 'where clause' in C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Model.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\BaseModel.php(524): CodeIgniter\Model->doFind(true, '1')
#6 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Authentication\AuthenticationBase.php(130): CodeIgniter\BaseModel->find('1')
#7 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
#8 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Filters\LoginFilter.php(29): Myth\Auth\Authentication\LocalAuthenticator->check()
#9 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Filters\Filters.php(173): Myth\Auth\Filters\LoginFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#10 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\CodeIgniter.php(455): CodeIgniter\Filters\Filters->run('mitra', 'before')
#11 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp7\htdocs\regsosek1206\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp7\\htdoc...')
#14 {main}
CRITICAL - 2022-12-14 17:13:36 --> Call to a member function getFirstRow() on bool
in SYSTEMPATH\Model.php on line 183.
 1 SYSTEMPATH\BaseModel.php(524): CodeIgniter\Model->doFind(true, '1')
 2 VENDORPATH\myth\auth\src\Authentication\AuthenticationBase.php(130): CodeIgniter\BaseModel->find('1')
 3 VENDORPATH\myth\auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
 4 VENDORPATH\myth\auth\src\Filters\LoginFilter.php(29): Myth\Auth\Authentication\LocalAuthenticator->check()
 5 SYSTEMPATH\Filters\Filters.php(173): Myth\Auth\Filters\LoginFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 6 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\Filters\Filters->run('mitra', 'before')
 7 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
ERROR - 2022-12-14 17:13:38 --> mysqli_sql_exception: Unknown column 'users.deleted_at' in 'where clause' in C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Model.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\BaseModel.php(524): CodeIgniter\Model->doFind(true, '1')
#6 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Authentication\AuthenticationBase.php(130): CodeIgniter\BaseModel->find('1')
#7 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
#8 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Filters\LoginFilter.php(29): Myth\Auth\Authentication\LocalAuthenticator->check()
#9 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Filters\Filters.php(173): Myth\Auth\Filters\LoginFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#10 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\CodeIgniter.php(455): CodeIgniter\Filters\Filters->run('mitra', 'before')
#11 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp7\htdocs\regsosek1206\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp7\\htdoc...')
#14 {main}
CRITICAL - 2022-12-14 17:13:38 --> Call to a member function getFirstRow() on bool
in SYSTEMPATH\Model.php on line 183.
 1 SYSTEMPATH\BaseModel.php(524): CodeIgniter\Model->doFind(true, '1')
 2 VENDORPATH\myth\auth\src\Authentication\AuthenticationBase.php(130): CodeIgniter\BaseModel->find('1')
 3 VENDORPATH\myth\auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
 4 VENDORPATH\myth\auth\src\Filters\LoginFilter.php(29): Myth\Auth\Authentication\LocalAuthenticator->check()
 5 SYSTEMPATH\Filters\Filters.php(173): Myth\Auth\Filters\LoginFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 6 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\Filters\Filters->run('mitra', 'before')
 7 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
ERROR - 2022-12-14 17:13:50 --> mysqli_sql_exception: Unknown column 'users.deleted_at' in 'where clause' in C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Model.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\BaseModel.php(524): CodeIgniter\Model->doFind(true, '1')
#6 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Authentication\AuthenticationBase.php(130): CodeIgniter\BaseModel->find('1')
#7 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
#8 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Filters\LoginFilter.php(29): Myth\Auth\Authentication\LocalAuthenticator->check()
#9 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Filters\Filters.php(173): Myth\Auth\Filters\LoginFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#10 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\CodeIgniter.php(455): CodeIgniter\Filters\Filters->run('regsosek2022', 'before')
#11 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp7\htdocs\regsosek1206\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp7\\htdoc...')
#14 {main}
CRITICAL - 2022-12-14 17:13:50 --> Call to a member function getFirstRow() on bool
in SYSTEMPATH\Model.php on line 183.
 1 SYSTEMPATH\BaseModel.php(524): CodeIgniter\Model->doFind(true, '1')
 2 VENDORPATH\myth\auth\src\Authentication\AuthenticationBase.php(130): CodeIgniter\BaseModel->find('1')
 3 VENDORPATH\myth\auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
 4 VENDORPATH\myth\auth\src\Filters\LoginFilter.php(29): Myth\Auth\Authentication\LocalAuthenticator->check()
 5 SYSTEMPATH\Filters\Filters.php(173): Myth\Auth\Filters\LoginFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 6 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\Filters\Filters->run('regsosek2022', 'before')
 7 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
ERROR - 2022-12-14 17:13:53 --> mysqli_sql_exception: Unknown column 'users.deleted_at' in 'where clause' in C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Model.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\BaseModel.php(524): CodeIgniter\Model->doFind(true, '1')
#6 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Authentication\AuthenticationBase.php(130): CodeIgniter\BaseModel->find('1')
#7 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
#8 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Filters\LoginFilter.php(29): Myth\Auth\Authentication\LocalAuthenticator->check()
#9 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Filters\Filters.php(173): Myth\Auth\Filters\LoginFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#10 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\CodeIgniter.php(455): CodeIgniter\Filters\Filters->run('/', 'before')
#11 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp7\htdocs\regsosek1206\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp7\\htdoc...')
#14 {main}
CRITICAL - 2022-12-14 17:13:53 --> Call to a member function getFirstRow() on bool
in SYSTEMPATH\Model.php on line 183.
 1 SYSTEMPATH\BaseModel.php(524): CodeIgniter\Model->doFind(true, '1')
 2 VENDORPATH\myth\auth\src\Authentication\AuthenticationBase.php(130): CodeIgniter\BaseModel->find('1')
 3 VENDORPATH\myth\auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
 4 VENDORPATH\myth\auth\src\Filters\LoginFilter.php(29): Myth\Auth\Authentication\LocalAuthenticator->check()
 5 SYSTEMPATH\Filters\Filters.php(173): Myth\Auth\Filters\LoginFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 6 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\Filters\Filters->run('C:\\', 'before')
 7 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
ERROR - 2022-12-14 17:15:55 --> mysqli_sql_exception: Unknown column 'users.deleted_at' in 'where clause' in C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Model.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\BaseModel.php(524): CodeIgniter\Model->doFind(true, '1')
#6 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Authentication\AuthenticationBase.php(130): CodeIgniter\BaseModel->find('1')
#7 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
#8 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Filters\LoginFilter.php(29): Myth\Auth\Authentication\LocalAuthenticator->check()
#9 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Filters\Filters.php(173): Myth\Auth\Filters\LoginFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#10 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\CodeIgniter.php(455): CodeIgniter\Filters\Filters->run('/', 'before')
#11 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp7\htdocs\regsosek1206\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp7\\htdoc...')
#14 {main}
CRITICAL - 2022-12-14 17:15:55 --> Call to a member function getFirstRow() on bool
in SYSTEMPATH\Model.php on line 183.
 1 SYSTEMPATH\BaseModel.php(524): CodeIgniter\Model->doFind(true, '1')
 2 VENDORPATH\myth\auth\src\Authentication\AuthenticationBase.php(130): CodeIgniter\BaseModel->find('1')
 3 VENDORPATH\myth\auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
 4 VENDORPATH\myth\auth\src\Filters\LoginFilter.php(29): Myth\Auth\Authentication\LocalAuthenticator->check()
 5 SYSTEMPATH\Filters\Filters.php(173): Myth\Auth\Filters\LoginFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 6 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\Filters\Filters->run('C:\\', 'before')
 7 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
ERROR - 2022-12-14 17:15:56 --> mysqli_sql_exception: Unknown column 'users.deleted_at' in 'where clause' in C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Model.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\BaseModel.php(524): CodeIgniter\Model->doFind(true, '1')
#6 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Authentication\AuthenticationBase.php(130): CodeIgniter\BaseModel->find('1')
#7 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
#8 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Controllers\AuthController.php(124): Myth\Auth\Authentication\LocalAuthenticator->check()
#9 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\CodeIgniter.php(927): Myth\Auth\Controllers\AuthController->register()
#10 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#11 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp7\htdocs\regsosek1206\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp7\\htdoc...')
#14 {main}
CRITICAL - 2022-12-14 17:15:56 --> Call to a member function getFirstRow() on bool
in SYSTEMPATH\Model.php on line 183.
 1 SYSTEMPATH\BaseModel.php(524): CodeIgniter\Model->doFind(true, '1')
 2 VENDORPATH\myth\auth\src\Authentication\AuthenticationBase.php(130): CodeIgniter\BaseModel->find('1')
 3 VENDORPATH\myth\auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
 4 VENDORPATH\myth\auth\src\Controllers\AuthController.php(124): Myth\Auth\Authentication\LocalAuthenticator->check()
 5 SYSTEMPATH\CodeIgniter.php(927): Myth\Auth\Controllers\AuthController->register()
 6 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 7 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
ERROR - 2022-12-14 17:15:57 --> mysqli_sql_exception: Unknown column 'users.deleted_at' in 'where clause' in C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Model.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\BaseModel.php(524): CodeIgniter\Model->doFind(true, '1')
#6 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Authentication\AuthenticationBase.php(130): CodeIgniter\BaseModel->find('1')
#7 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
#8 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Filters\LoginFilter.php(29): Myth\Auth\Authentication\LocalAuthenticator->check()
#9 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Filters\Filters.php(173): Myth\Auth\Filters\LoginFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#10 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\CodeIgniter.php(455): CodeIgniter\Filters\Filters->run('/', 'before')
#11 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp7\htdocs\regsosek1206\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp7\\htdoc...')
#14 {main}
CRITICAL - 2022-12-14 17:15:57 --> Call to a member function getFirstRow() on bool
in SYSTEMPATH\Model.php on line 183.
 1 SYSTEMPATH\BaseModel.php(524): CodeIgniter\Model->doFind(true, '1')
 2 VENDORPATH\myth\auth\src\Authentication\AuthenticationBase.php(130): CodeIgniter\BaseModel->find('1')
 3 VENDORPATH\myth\auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
 4 VENDORPATH\myth\auth\src\Filters\LoginFilter.php(29): Myth\Auth\Authentication\LocalAuthenticator->check()
 5 SYSTEMPATH\Filters\Filters.php(173): Myth\Auth\Filters\LoginFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 6 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\Filters\Filters->run('C:\\', 'before')
 7 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
ERROR - 2022-12-14 17:16:05 --> mysqli_sql_exception: Unknown column 'users.deleted_at' in 'where clause' in C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Model.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\BaseModel.php(524): CodeIgniter\Model->doFind(true, '1')
#6 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Authentication\AuthenticationBase.php(130): CodeIgniter\BaseModel->find('1')
#7 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
#8 C:\xampp7\htdocs\regsosek1206\vendor\myth\auth\src\Filters\LoginFilter.php(29): Myth\Auth\Authentication\LocalAuthenticator->check()
#9 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Filters\Filters.php(173): Myth\Auth\Filters\LoginFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#10 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\CodeIgniter.php(455): CodeIgniter\Filters\Filters->run('/', 'before')
#11 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp7\htdocs\regsosek1206\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp7\htdocs\regsosek1206\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp7\\htdoc...')
#14 {main}
CRITICAL - 2022-12-14 17:16:05 --> Call to a member function getFirstRow() on bool
in SYSTEMPATH\Model.php on line 183.
 1 SYSTEMPATH\BaseModel.php(524): CodeIgniter\Model->doFind(true, '1')
 2 VENDORPATH\myth\auth\src\Authentication\AuthenticationBase.php(130): CodeIgniter\BaseModel->find('1')
 3 VENDORPATH\myth\auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
 4 VENDORPATH\myth\auth\src\Filters\LoginFilter.php(29): Myth\Auth\Authentication\LocalAuthenticator->check()
 5 SYSTEMPATH\Filters\Filters.php(173): Myth\Auth\Filters\LoginFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 6 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\Filters\Filters->run('C:\\', 'before')
 7 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-14 19:52:51 --> Undefined variable $title
in APPPATH\Views\templates\header.php on line 11.
 1 APPPATH\Views\templates\header.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $title', 'APPPATH\\Views\\templates\\header.php', 11)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\templates\\header.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('templates/header', [], true)
 5 APPPATH\Controllers\Mitra.php(20): view('templates/header')
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Mitra->tambah()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mitra))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-14 19:53:01 --> Undefined variable $title
in APPPATH\Views\templates\header.php on line 11.
 1 APPPATH\Views\templates\header.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $title', 'APPPATH\\Views\\templates\\header.php', 11)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\templates\\header.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('templates/header', [], true)
 5 APPPATH\Controllers\Mitra.php(20): view('templates/header')
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Mitra->tambah()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mitra))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-14 19:54:33 --> Undefined variable $title
in APPPATH\Views\templates\header.php on line 11.
 1 APPPATH\Views\templates\header.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $title', 'APPPATH\\Views\\templates\\header.php', 11)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\templates\\header.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('templates/header', [], true)
 5 APPPATH\Controllers\Mitra.php(20): view('templates/header')
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Mitra->tambah()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mitra))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-14 19:54:56 --> Undefined variable $title
in APPPATH\Views\templates\header.php on line 11.
 1 APPPATH\Views\templates\header.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $title', 'APPPATH\\Views\\templates\\header.php', 11)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\templates\\header.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('templates/header', [], true)
 5 APPPATH\Controllers\Mitra.php(20): view('templates/header')
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Mitra->tambah()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mitra))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-14 20:37:42 --> The route for "/mitra/tambah" cannot be found.
in SYSTEMPATH\Helpers\url_helper.php on line 543.
 1 SYSTEMPATH\Helpers\url_helper.php(543): CodeIgniter\Router\Exceptions\RouterException::forInvalidRoute('/mitra/tambah')
 2 APPPATH\Views\mitra\tambah.php(12): url_to('/mitra/tambah')
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\mitra\\tambah.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('mitra/tambah', [], true)
 6 APPPATH\Controllers\Mitra.php(25): view('mitra/tambah')
 7 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Mitra->tambah()
 8 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mitra))
 9 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-14 20:37:56 --> The route for "/mitra/tambah" cannot be found.
in SYSTEMPATH\Helpers\url_helper.php on line 543.
 1 SYSTEMPATH\Helpers\url_helper.php(543): CodeIgniter\Router\Exceptions\RouterException::forInvalidRoute('/mitra/tambah')
 2 APPPATH\Views\mitra\tambah.php(12): url_to('/mitra/tambah')
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\mitra\\tambah.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('mitra/tambah', [], true)
 6 APPPATH\Controllers\Mitra.php(25): view('mitra/tambah')
 7 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Mitra->tambah()
 8 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mitra))
 9 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-14 20:38:00 --> The route for "/mitra/tambah" cannot be found.
in SYSTEMPATH\Helpers\url_helper.php on line 543.
 1 SYSTEMPATH\Helpers\url_helper.php(543): CodeIgniter\Router\Exceptions\RouterException::forInvalidRoute('/mitra/tambah')
 2 APPPATH\Views\mitra\tambah.php(12): url_to('/mitra/tambah')
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\mitra\\tambah.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('mitra/tambah', [], true)
 6 APPPATH\Controllers\Mitra.php(25): view('mitra/tambah')
 7 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Mitra->tambah()
 8 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mitra))
 9 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-14 20:38:29 --> The route for "/mitra/tambah" cannot be found.
in SYSTEMPATH\Helpers\url_helper.php on line 543.
 1 SYSTEMPATH\Helpers\url_helper.php(543): CodeIgniter\Router\Exceptions\RouterException::forInvalidRoute('/mitra/tambah')
 2 APPPATH\Views\mitra\tambah.php(12): url_to('/mitra/tambah')
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\mitra\\tambah.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('mitra/tambah', [], true)
 6 APPPATH\Controllers\Mitra.php(25): view('mitra/tambah')
 7 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Mitra->tambah()
 8 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mitra))
 9 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-14 20:38:30 --> The route for "/mitra/tambah" cannot be found.
in SYSTEMPATH\Helpers\url_helper.php on line 543.
 1 SYSTEMPATH\Helpers\url_helper.php(543): CodeIgniter\Router\Exceptions\RouterException::forInvalidRoute('/mitra/tambah')
 2 APPPATH\Views\mitra\tambah.php(12): url_to('/mitra/tambah')
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\mitra\\tambah.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('mitra/tambah', [], true)
 6 APPPATH\Controllers\Mitra.php(25): view('mitra/tambah')
 7 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Mitra->tambah()
 8 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mitra))
 9 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-14 20:38:30 --> The route for "/mitra/tambah" cannot be found.
in SYSTEMPATH\Helpers\url_helper.php on line 543.
 1 SYSTEMPATH\Helpers\url_helper.php(543): CodeIgniter\Router\Exceptions\RouterException::forInvalidRoute('/mitra/tambah')
 2 APPPATH\Views\mitra\tambah.php(12): url_to('/mitra/tambah')
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\mitra\\tambah.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('mitra/tambah', [], true)
 6 APPPATH\Controllers\Mitra.php(25): view('mitra/tambah')
 7 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Mitra->tambah()
 8 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mitra))
 9 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-14 20:38:31 --> The route for "/mitra/tambah" cannot be found.
in SYSTEMPATH\Helpers\url_helper.php on line 543.
 1 SYSTEMPATH\Helpers\url_helper.php(543): CodeIgniter\Router\Exceptions\RouterException::forInvalidRoute('/mitra/tambah')
 2 APPPATH\Views\mitra\tambah.php(12): url_to('/mitra/tambah')
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\mitra\\tambah.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('mitra/tambah', [], true)
 6 APPPATH\Controllers\Mitra.php(25): view('mitra/tambah')
 7 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Mitra->tambah()
 8 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mitra))
 9 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-14 20:38:36 --> The route for "/mitra/tambah" cannot be found.
in SYSTEMPATH\Helpers\url_helper.php on line 543.
 1 SYSTEMPATH\Helpers\url_helper.php(543): CodeIgniter\Router\Exceptions\RouterException::forInvalidRoute('/mitra/tambah')
 2 APPPATH\Views\mitra\tambah.php(12): url_to('/mitra/tambah')
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\mitra\\tambah.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('mitra/tambah', [], true)
 6 APPPATH\Controllers\Mitra.php(25): view('mitra/tambah')
 7 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Mitra->tambah()
 8 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mitra))
 9 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-14 20:38:59 --> The route for "/mitra/tambah" cannot be found.
in SYSTEMPATH\Helpers\url_helper.php on line 543.
 1 SYSTEMPATH\Helpers\url_helper.php(543): CodeIgniter\Router\Exceptions\RouterException::forInvalidRoute('/mitra/tambah')
 2 APPPATH\Views\mitra\tambah.php(12): url_to('/mitra/tambah')
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\mitra\\tambah.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('mitra/tambah', [], true)
 6 APPPATH\Controllers\Mitra.php(25): view('mitra/tambah')
 7 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Mitra->tambah()
 8 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mitra))
 9 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-14 20:39:00 --> The route for "/mitra/tambah" cannot be found.
in SYSTEMPATH\Helpers\url_helper.php on line 543.
 1 SYSTEMPATH\Helpers\url_helper.php(543): CodeIgniter\Router\Exceptions\RouterException::forInvalidRoute('/mitra/tambah')
 2 APPPATH\Views\mitra\tambah.php(12): url_to('/mitra/tambah')
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\mitra\\tambah.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('mitra/tambah', [], true)
 6 APPPATH\Controllers\Mitra.php(25): view('mitra/tambah')
 7 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Mitra->tambah()
 8 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mitra))
 9 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-14 20:39:01 --> The route for "/mitra/tambah" cannot be found.
in SYSTEMPATH\Helpers\url_helper.php on line 543.
 1 SYSTEMPATH\Helpers\url_helper.php(543): CodeIgniter\Router\Exceptions\RouterException::forInvalidRoute('/mitra/tambah')
 2 APPPATH\Views\mitra\tambah.php(12): url_to('/mitra/tambah')
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\mitra\\tambah.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('mitra/tambah', [], true)
 6 APPPATH\Controllers\Mitra.php(25): view('mitra/tambah')
 7 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Mitra->tambah()
 8 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mitra))
 9 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-14 20:39:06 --> The route for "/mitra/tambah" cannot be found.
in SYSTEMPATH\Helpers\url_helper.php on line 543.
 1 SYSTEMPATH\Helpers\url_helper.php(543): CodeIgniter\Router\Exceptions\RouterException::forInvalidRoute('/mitra/tambah')
 2 APPPATH\Views\mitra\tambah.php(12): url_to('/mitra/tambah')
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\mitra\\tambah.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('mitra/tambah', [], true)
 6 APPPATH\Controllers\Mitra.php(25): view('mitra/tambah')
 7 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Mitra->tambah()
 8 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mitra))
 9 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-14 20:39:19 --> The route for "/mitra/tambah" cannot be found.
in SYSTEMPATH\Helpers\url_helper.php on line 543.
 1 SYSTEMPATH\Helpers\url_helper.php(543): CodeIgniter\Router\Exceptions\RouterException::forInvalidRoute('/mitra/tambah')
 2 APPPATH\Views\mitra\tambah.php(12): url_to('/mitra/tambah')
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\mitra\\tambah.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('mitra/tambah', [], true)
 6 APPPATH\Controllers\Mitra.php(25): view('mitra/tambah')
 7 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Mitra->tambah()
 8 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mitra))
 9 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-14 20:39:41 --> The route for "/mitra/tambah" cannot be found.
in SYSTEMPATH\Helpers\url_helper.php on line 543.
 1 SYSTEMPATH\Helpers\url_helper.php(543): CodeIgniter\Router\Exceptions\RouterException::forInvalidRoute('/mitra/tambah')
 2 APPPATH\Views\mitra\tambah.php(12): url_to('/mitra/tambah')
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\mitra\\tambah.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('mitra/tambah', [], true)
 6 APPPATH\Controllers\Mitra.php(25): view('mitra/tambah')
 7 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Mitra->tambah()
 8 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mitra))
 9 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
